source/comply/data-retention-policy.rst

```diff
@@ -10,18 +10,35 @@ In Mattermost Enterprise E20, you can set a custom policy to manage how long mes
 .. warning:: 
   Once a message or a file is deleted, the action is irreversible. Please use caution when setting up a custom data retention policy.
 
-Configuring a Data Retention Policy
-------------------------------------
+Configuring a Global Data Retention Policy
+-------------------------------------------
 
-To set a custom data retention policy:
+To set a global data retention policy:
 
 1. Go to **System Console > Compliance > Data Retention Policy**.
-2. Select a **Message Retention** option. When a time is specified, messages, including file attachments, older than the duration you set will be deleted at the specified time. The minimum retention period is one day.
-3. Select a **File Retention** option. When a time is specified uploaded files which are older than the duration you set will be deleted from your file storage system (either from your local disk or your Amazon S3 service as specified in **System Console > Environment > File Storage** at the specified time. The minimum retention period is one day.
-4. Set the start time of the daily scheduled data retention job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.
+2. Select edit from the menu presented to the right of the **Global retention policy** table. 
+3. Select a **Channel & direct message retention** option from the dropdown. Specify the number of days or years to keep channel and direct messages. When a time is specified, messages, including file attachments, older than the duration you set will be deleted at the specified time. The minimum retention period is one day.
+3. Select a **File retention** option from the dropdown. Specify the number of days or ears to keep files. When a time is specified uploaded files which are older than the duration you set will be deleted from your file storage system (either from your local disk or your Amazon S3 service as specified in **System Console > Environment > File Storage** at the specified time. The minimum retention period is one day.
```

Comment:
```suggestion
3. Specify a global retention policy for files by selecting a **File retention** option from the dropdown, then specifying how long to keep uploaded files. When a time is specified, uploaded files older than the duration you set will be deleted from your file storage system (either from your local disk or your Amazon S3 service as specified in **System Console > Environment > File Storage**) at the specified time. The minimum retention period is one day.
```

Reply:
We appear to be making a distinction between file attachments on a channel or DM message versus uploaded files. Is this intentional? If so, we may want to clearly emphasize the difference between the two types of "files" and under what conditions each global retention option will work.

Reply:
the number 3 is used for two different bullet.

Reply:
> We appear to be making a distinction between file attachments on a channel or DM message versus uploaded files. Is this intentional? If so, we may want to clearly emphasize the difference between the two types of "files" and under what conditions each global retention option will work.

Every file in the Channels product is attached to a post. The global file policy deletes all files (regardless of whether they're in a direct message, private, or public channel).